pom.xml

```
@@ -96,6 +96,11 @@
 			<artifactId>jjwt-api</artifactId>
 			<version>${jjwt.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-docker-compose</artifactId>
+			<version>3.1.1</version>
+		</dependency>
```

1. Do you need this dependency? It forces your application to be run via docker compose. 
2. Let spring boot manage version.
```suggestion
		<dependency>
			<groupId>org.springframework.boot</groupId>
			<artifactId>spring-boot-docker-compose</artifactId>
		</dependency>
```